objects/src/accounts/component/template/storage/value_type.rs

```diff
@@ -234,6 +256,18 @@ impl Deserializable for FeltRepresentation {
     }
 }
 
+impl core::fmt::Display for FeltRepresentation {
+    fn fmt(&self, f: &mut core::fmt::Formatter<'_>) -> core::fmt::Result {
+        match self {
+            FeltRepresentation::Hexadecimal(base_element) => f.write_str(&base_element.to_string()),
+            FeltRepresentation::Decimal(base_element) => f.write_str(&base_element.to_string()),
+            FeltRepresentation::Template(storage_placeholder) => {
+                f.write_str(&storage_placeholder.to_string())
+            },
```

Comment:
Nit: We can avoid allocations here as well, e.g. with `f.write_fmt(format_args!("{}", base_element))`.